### ISSUE TITLE
Blank page after login

### ISSUE THREAD

**Issue by astor99** [lirantal/daloradius#118]
After the login administrator/radius, I have a blank page. I'm trying to fix it but I don't know how to do that.
I have:

Centos 7
PHP 7.1
MySQL Ver 15.1 Distrib 5.5.60-MariaDB
Freeradius 3

I'm getting this error in Nginx log file:

`2019/08/07 06:28:06 [error] 767#0: *80 FastCGI sent in stderr: "PHP message: PHP Warning:  session_start(): open(/var/lib/php/session/sess_677f87875f6edf56164724343b45d0f6c, O_RDWR) failed: Permission denied (13) in /var/www/daloradius/library/checklogin.php on line 27
PHP message: PHP Warning:  session_start(): Failed to read session data: files (path: /var/lib/php/session) in /var/www/daloradius/library/checklogin.php on line 27" while reading response header from upstream, client: 212.194.109.83, server: radius.xxx.com, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://unix:/var/run/php-fpm/php-fpm.sock:", host: "radius.xxx.com", referrer: "http://radius.xxx.com/login.php"
2019/08/07 06:28:06 [error] 767#0: *80 FastCGI sent in stderr: "PHP message: PHP Warning:  A non-numeric value encountered in /var/www/daloradius/lang/en.php on line 1409
PHP message: PHP Warning:  A non-numeric value encountered in /var/www/daloradius/lang/en.php on line 1410
PHP message: PHP Warning:  A non-numeric value encountered in /var/www/daloradius/lang/en.php on line 1411
PHP message: PHP Warning:  A non-numeric value encountered in /var/www/daloradius/lang/en.php on line 1412" while reading response header from upstream, client: 12.194.109.83, server: radius.xxx.com, request: "GET /login.php HTTP/1.1", upstream: "fastcgi://unix:/var/run/php-fpm/php-fpm.sock:", host: "radius.xxx.com", referrer: "http://radius.xxx.com/login.php"`

Something to change in permissions?
Can you support me, please?

**Comment by MiguelVis** [comment]
Hi @astor99.

Yes, it seems a permissions issue regarding the directory ´/var/lib/php/session/´ where sessions are stored.

You could find of interest the following link:

[https://pathtogeek.com/how-to-upgrade-php-5-6-to-7-1-in-centos-7-linux](https://pathtogeek.com/how-to-upgrade-php-5-6-to-7-1-in-centos-7-linux)

where you will find how to solve this issue, by executing on the command line the following commands:

````
# cd /var/run/php/
# chown -R root:nginx *
````

Please, confirm you have solved the issue, after following these instructions, in order to close it.

**Comment by astor99** [comment]
Thanks for your help!!! It finally worked !!
I did that:

```
# cd /var/lib/php/session/
# chown -R root:nginx *
```
You saved me a lot of time.

**Comment by MiguelVis** [comment]
Hi @astor99.

You're welcome.

In turn, thank you for pasting here your error log, because it helped me to discover a bug in some language translation files, now solved with PR #119:

`PHP Warning: A non-numeric value encountered in /var/www/daloradius/lang/en.php`
